src/app/resources/rate-resources/resources-rate.component.ts

```diff
@@ -0,0 +1,126 @@
+import { switchMap } from 'rxjs/operators';
+import { Component, OnInit } from '@angular/core';
+import { CouchService } from '../../shared/couchdb.service';
+import { UserService } from '../../shared/user.service';
+
+import { ActivatedRoute, ParamMap } from '@angular/router';
+import { DomSanitizer } from '@angular/platform-browser';
+import { Router } from '@angular/router';
```

Comment:
this can merge with line 6

Reply:
@lmmrssa All fixed https://github.com/ole-vi/planet/pull/270/commits/9540e845997088564010b8e9996cb37f1b38fb71